### PR TITLE
Update for new Tribler configuration system

### PR DESCRIPTION
**Pull request by vandenheuvel**
This PR needs to be merged once https://github.com/Tribler/tribler/pull/2841 is merged.

### CODE REVIEW

**Comment by qstokkink** [comment]
ok to test

**Review by devos50 — APPROVED** [review]
I think this PR is ok.

File: gumby/modules/base_dispersy_module.py [hunk 1]
```diff
@@ -60,26 +60,24 @@ def setup_session_config(self):
...
-        config.set_enable_torrent_search(False)
-        config.set_enable_channel_search(False)
-        config.set_videoserver_enabled(False)
+        config.set_torrent_checking_enabled(False)
+        config.set_megacache_enabled(False)
+        config.set_dispersy_enabled(True)
+        config.set_mainline_dht_enabled(False)
+        config.set_torrent_collecting_enabled(False)
+        config.set_libtorrent_enabled(False)
+        config.set_torrent_collecting_enabled(False)
```
Comment (qstokkink) [inline gumby/modules/base_dispersy_module.py]: Duplicate of 2 lines above.

Comment (vandenheuvel) [reply gumby/modules/base_dispersy_module.py]: Nice one, thanks.

**Review by qstokkink — APPROVED** [review]

**Comment by devos50** [comment]
@vandenheuvel please let me know when the multi chain experiment works. @qstokkink are we then ready to merge this?

**Comment by vandenheuvel** [comment]
@devos50 it works. However, please note that I decided to put the pull request for Tribler on WIP again.

**Comment by qstokkink** [comment]
@devos50 I would say so, except @vandenheuvel what's the hold up?

**Pull request merged**